### DISCUSSION TITLE
[Enhancement]: Adding breadcrumb lines.

### DISCUSSION THREAD

**Discussion by 1-Mythical** [winstxnhdw/lc-hax#109] [Ideas]
I am unsure if the term "breadcrumb" is used anywhere besides some Minecraft hacked clients, but what breadcrumbs do is leave a trail like some type of rainbow line or colored line that will leave a trail. You can then follow this trail again later to know where to go. If ESP won't display fire exits or the entrance, then why not add this as a keybind toggle.

**Comment by winstxnhdw** [comment]
I think I'll get ESP to show the exits.
